packages/story-editor/src/types.js

```diff
@@ -63,6 +63,7 @@ StoryPropTypes.story = PropTypes.shape({
   currentStoryStyles: PropTypes.object,
   autoAdvance: PropTypes.bool,
   defaultPageDuration: PropTypes.number,
+  backgroundAudio: PropTypes.string,
```

Comment:
Let's define a audioResource type here. And use it in page and story, instead of this string.